build_qt5.bat

```diff
@@ -1,7 +1,10 @@
 SETLOCAL
 echo.
 echo ---- Building Qt5 ----
-SET QT5_PATH=qt-everywhere-opensource-src-5.7.1
+
+REM Qt5 paths can get really long. We use an abbreviated folder name to prevent hitting 
+REM Windows path length limits.
+SET QT5_PATH=Q5101
```

Comment:
Can't this be a _little_ more descriptive? e.g. `Qt5.10.1`?

Reply:
Gotta save every byte :). It failed with `Qt-5.10.1` so we're close to the limit. I think we can shrink the Jenkins workspace name though, which is `bs-2.1.0-win` or something.